src/bun.js/base.zig

```diff
@@ -553,7 +553,7 @@ pub const ArrayBuffer = extern struct {
     ///    new ArrayBuffer(view.buffer, view.byteOffset, view.byteLength)
     /// ```
     pub inline fn byteSlice(this: *const @This()) []u8 {
-        return this.ptr[this.offset .. this.offset + this.byte_len];
+        return this.ptr[this.offset..][0..this.byte_len];
```

Comment:
What was the reason for this change? Zig was/is missing safety checks in `[offset...][0..len]`

Reply:
fixed in https://github.com/ziglang/zig/pull/19374 which made it into both 0.12 and 0.13